--- a/configure	2024-04-05 01:22:59.000000000 +0200
+++ b/configure	2024-07-18 09:38:36.313412981 +0200
@@ -2293,6 +2293,7 @@
     opencv2_core_core_c_h
     OpenGL_gl3_h
     poll_h
+    sys_auxv_h
     pthread_np_h
     sys_param_h
     sys_resource_h
@@ -6612,6 +6613,7 @@
 check_func_headers VideoToolbox/VideoToolbox.h VTPixelRotationSessionCreate -framework VideoToolbox
 check_headers windows.h
 check_headers asm/types.h
+check_headers sys/auxv.h
 
 # it seems there are versions of clang in some distros that try to use the
 # gcc headers, which explodes for stdatomic
--- a/libavutil/mips/cpu.c 2023-11-10 00:38:51.000000000 +0100
+++ b/libavutil/mips/cpu.c  2024-07-18 09:42:40.466458380 +0200
@@ -19,7 +19,7 @@
 #include "libavutil/cpu.h"
 #include "libavutil/cpu_internal.h"
 #include "config.h"
-#if defined __linux__ || defined __ANDROID__
+#if (defined __linux__ || defined __ANDROID__) && HAVE_SYS_AUXV_H
 #include <stdint.h>
 #include <stdio.h>
 #include <string.h>
@@ -28,7 +28,7 @@
 #include "libavutil/avstring.h"
 #endif
 
-#if defined __linux__ || defined __ANDROID__
+#if (defined __linux__ || defined __ANDROID__) && HAVE_SYS_AUXV_H
 
 #define HWCAP_LOONGSON_CPUCFG (1 << 14)
 
@@ -105,7 +105,7 @@
 
 int ff_get_cpu_flags_mips(void)
 {
-#if defined __linux__ || defined __ANDROID__
+#if (defined __linux__ || defined __ANDROID__) && HAVE_SYS_AUXV_H
     if (cpucfg_available())
         return cpu_flags_cpucfg();
     else
